core/commands.c

```diff
@@ -43,13 +43,15 @@ command_t commands[];
 
 static int helpCommand(size_t argc, char *argv[], size_t outbufc,
                        char outbuf[]) {
+  pod_t *pod = get_pod();
   int count =
       snprintf(&outbuf[0], outbufc, "%s",
                "Pod CLI " POD_CLI_VERSION_STR ". Copyright " POD_COPY_YEAR
                " " POD_COPY_OWNER "\n" POD_CREDITS
                "This tool allows you to control various aspects of the pod\n"
-               " - TCP:" __XSTR__(CMD_SVR_PORT) "\n - STDIN\n\n"
-                                                "Available Commands:\n");
+               " - TCP:" __XSTR__(
+                   get_value(&pod->command_port)) "\n - STDIN\n\n"
+                                                  "Available Commands:\n");
```

Comment:
`__XSTR__` is meant to convert the value of a preprocessor macro to a string.  i.e. if `CMD_SVR_PORT` is defined as `7779` then `__XSTR__(CMD_SVR_PORT)` will pre-process to `"7779"` and thus the preprocessed C will look like:

```c
"...aspects of the pod\n" " - TCP:" "7779" "\n - STDIN\n\n" ...
```
(The compiler will automatically concatenate the string literals into a single string)

Now... `get_value(&pod->command_port)` is not a preprocessor macro, and therefore is not known at compile time.  Therefore you need to format the string at runtime using `snprintf`.

Conveniently, there is already an `snprintf` call wrapping this string to format in some other variables... although those variables are kindof messed up.  This should read something like this:

```c
int count =
      snprintf(&outbuf[0], outbufc, "Pod CLI " POD_CLI_VERSION_STR ". Copyright " POD_COPY_YEAR
               " " POD_COPY_OWNER "\n" POD_CREDITS
               "This tool allows you to control various aspects of the pod\n"
               " - TCP: %d\n - STDIN\n\n"
                                                "Available Commands:\n", get_value(&pod->command_port)));
```

Reply:
This will end up printing 

```
...
 - TCP: get_value(&pod->command_port)
...
```